tests/integration/backend_test.py

```diff
@@ -1106,12 +1106,22 @@ def test_old_handle(
     assert (r0.get_shots() == r1.get_shots()).all()
 
 
+@pytest.mark.parametrize(
```

Comment:
Should we parametrize the other failing test as well?

Reply:
done